docs/requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -115,6 +115,32 @@ Added an `event.completed` option to format a message in a Smart Alerts dialog u
 
 ---
 
+### Spam reporting improvements
+
+#### Suppress the preprocessing dialog
+
+Added the [NeverShowAgainOption](/javascript/api/manifest/preprocessingdialog?view=common-js-preview&preserve-view=true#child-elements) element as a child of the **\<PreProcessingDialog\>** element. This element adds a "Don't show me this message again" checkbox to the preprocessing dialog to prevent it from appearing again.
```

Comment:
Consider also having this link in Outlook context

```suggestion
Added the [NeverShowAgainOption](/javascript/api/manifest/preprocessingdialog?view=outlook-js-preview&preserve-view=true#child-elements) element as a child of the **\<PreProcessingDialog\>** element. This element adds a "Don't show me this message again" checkbox to the preprocessing dialog to prevent it from appearing again.
```